snakes_cafe.py

```diff
@@ -0,0 +1,47 @@
+import sys
+
+print('***************************************************')
+print('**        Welcome to the Snakes Cafe!            **')
+print('**        Please see our menu below.             **')
+print('**')
+print('**         To quit at any time, type "quit"      **')
+print('***************************************************\n')
+
+print('Appetizers')
+print('---------')
+print('Wings')
+print('Cookies')
+print('Spring Rolls\n')
+
+print('Entrees')
+print('--------')
+print('Salmon')
+print('Steak')
+print('Meat Tornado')
+print('A Literal Garden\n')
+
+print('Desserts')
+print('--------')
+print('Ice cream')
+print('Cake')
+print('Pie\n')
+
+print('Drinks')
+print('------')
+print('Coffee')
+print('Tea')
+print('Blood of the Innocent\n')
+
+
+print('***************************************************')
+print('**      What would you like to order?            **')
+print('***************************************************')
+
+
+
+while True:
+    order = input('>')
+    if order == 'quit':
+        quit()
+    elif:
+        print(f' ** {order} order has been added to your order **')
```

Comment:
There has to be some kind of logic that checks to confirm that the user's order is available in a menu. Further, if the user has already ordered something (like wings) and they order it again, the print statement should indicate the number of that item in the order. See the comp images in the lab markdown and Scott's demo for reference.